src/messages.js

```diff
@@ -74,7 +74,8 @@ var errors = {
   E056: "'{a}' was used before it was declared, which is illegal for '{b}' variables.",
   E057: "Invalid meta property: '{a}.{b}'.",
   E058: "Missing semicolon.",
-  E059: "Incompatible values for the '{a}' and '{b}' linting options."
+  E059: "Incompatible values for the '{a}' and '{b}' linting options.",
+  E060: "Literals and unary operators cannot be used after the `instanceof` operator."
```

Comment:
We should say, "Non-callable values" to more precisely describe the error.
Also, let's say "as the second operand to instanceof".